packages-exp/performance-exp/src/services/initialization_service.ts

```diff
@@ -41,10 +45,12 @@ export function isPerfInitialized(): boolean {
   return initializationStatus === InitializationStatus.initialized;
 }
 
-function initializePerf(): Promise<void> {
+function initializePerf(
+  performanceController: PerformanceController
+): Promise<void> {
   return getDocumentReadyComplete()
-    .then(() => getIidPromise())
-    .then(iid => getConfig(iid))
+    .then(() => getIidPromise(performanceController.installations))
```

Comment:
Do we need to send the whole performanceController down the chain to be able to use installations? How about having a smaller subset of fields in an object without everything else that the performanceController has?

Reply:
From here we only need the installations, but following the chain from the call below, we need both the installations as well as the FirebaseApp for the RemoteConfig service.

If you think it makes sense, I can create an object with those two fields (which is the only reason we pass the PerformanceController around). I'm not sure if it would make any difference though: what is being passed around is a pointer, so I don't think it would make any difference to pass around a pointer to the PerformanceController vs a pointer to a different object.

Let me know your thoughts.

Reply:
I think it would be cleaner to send down only the needed information as opposed to the main class. Having said that, I do not think this is a p1. More of a clean up that we can do later.

Reply:
Sounds good. Will leave it as is for now.